x11-misc/j4-dmenu-desktop/j4-dmenu-desktop-2.16.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+MY_PV="r${PV}"
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="A fast desktop menu"
+HOMEPAGE="https://github.com/enkore/j4-dmenu-desktop"
+SRC_URI="https://github.com/enkore/j4-dmenu-desktop/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Any reason not to name it `${MY_P}.tar.gz`? That would be more consistent to how it's unpacked.

Reply:
No reason.